### PR TITLE
Add cli command contao:migrate

### PR DESCRIPTION
**Pull request by fkaminski**
Some translations are not so elegant. Suggestions are welcome ...

### CODE REVIEW

File: docs/manual/cli/migrate.de.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+
+{{< version "4.9" >}}
+
+Mit diesem Befehl können Datenbank-Updates und Migrationen nach einer Neuinstallation, einem Contao Update oder einer 
+Erweiterung durchgeführt werden. Die Migrationen, die ausgeführt werden, sind Contao Update-Skripte, registrierte Migrationen von Erweiterungen, 
+Legacy `runonce.php`-Dateien und das Datenbank-Update.
+
+
+```sh
```
Comment (netzarbeiter) [inline docs/manual/cli/migrate.de.md]:
```suggestion
```bash
```

File: docs/manual/cli/migrate.de.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+$> php vendor/bin/contao-console contao:migrate [options]
```
Comment (netzarbeiter) [inline docs/manual/cli/migrate.de.md]:
```suggestion
php vendor/bin/contao-console contao:migrate [options]
```

File: docs/manual/cli/migrate.en.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+
+{{< version "4.9" >}}
+
+With this command you can perform database updates and migrations after a new installation or update of Contao or an 
+extension. The migrations that get executed are update scripts of Contao, registered migrations of extensions, 
+legacy `runonce.php` files and the database update.
+
+```sh
```
Comment (netzarbeiter) [inline docs/manual/cli/migrate.en.md]:
```suggestion
```bash
```

File: docs/manual/cli/migrate.en.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+$> php vendor/bin/contao-console contao:migrate [options]
```
Comment (netzarbeiter) [inline docs/manual/cli/migrate.en.md]:
```suggestion
php vendor/bin/contao-console contao:migrate [options]
```

File: docs/manual/cli/migrate.de.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+```
+
+| Option | Beschreibung |
+| --- | --- |
+| `--with-deletes`   | Führt alle Datenbankmigrationen einschließlich `DROP`-Abfragen aus. |
+| `--schema-only`    | Führt nur die Migration des Datenbankschemas aus. Update-Skripte, registrierte Migrationen und `runonce.php`-Dateien werden übersprungen. |
+| `--no-interaction` | Wenn diese Option aktiviert ist, werden alle Bestätigungsfragen automatisch mit "Ja" beantwortet. Dies ist nützlich, wenn die Migrationen in einem automatisierten System durchgeführt werden sollen. |
```
Comment (netzarbeiter) [inline docs/manual/cli/migrate.de.md]:
```suggestion
| `--no-interaction` | Wenn diese Option aktiviert ist, werden alle Bestätigungsfragen automatisch mit »Ja« beantwortet. Dies ist nützlich, wenn die Migrationen in einem automatisierten System durchgeführt werden sollen. |
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Franko

File: docs/manual/cli/migrate.de.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+Legacy `runonce.php`-Dateien und das Datenbank-Update.
+
+
+```bash
+php vendor/bin/contao-console contao:migrate [options]
+```
+
+| Option | Beschreibung |
+| --- | --- |
+| `--with-deletes`   | Führt alle Datenbankmigrationen einschließlich `DROP`-Abfragen aus. |
```
Comment (ausi) [inline docs/manual/cli/migrate.de.md]:
```suggestion
| `--with-deletes`   | Führt alle Datenbankmigrationen einschließlich `DROP`-Befehlen aus. |
```

File: docs/manual/cli/migrate.en.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+```bash
+php vendor/bin/contao-console contao:migrate [options]
+```
+
+| Option             | Description |
+|--------------------|-------------|
+| `--with-deletes`   | Executes all database migrations including `DROP` queries.|
+| `--schema-only`    | Executes database schema migration only. Update scripts, registered migrations and `runonce.php` files get skipped.|
+| `--no-interaction` | With this option enabled all confirmation questions are automatically answered with “yes”. This is useful if you want to execute the migrations in an automated system.|
```
Comment (fritzmg) [inline docs/manual/cli/migrate.en.md]:
```suggestion
| `--migrations-only` | Only executes the migrations without updating the database tables and fields. |
| `--dry-run` | Shows pending migrations and schema updates without actually executing them. |
| `--no-interaction` | With this option enabled all confirmation questions are automatically answered with “yes”. This is useful if you want to execute the migrations in an automated system. |
| `--no-backup` | {{< version-tag "4.13" }} Disable the [database backup][DatabaseBackup] which is created by default before executing the migrations. |

[DatabaseBackup]: /en/cli/db-backups/
```

**Review by fritzmg — CHANGES_REQUESTED** [review]

File: docs/manual/cli/migrate.en.md (hunk 7 above)
Comment (fkaminski) [reply docs/manual/cli/migrate.en.md]: From https://docs.contao.org/dev/reference/commands/#contao-migrate
Why delete the option --no-interaction?

Comment (fkaminski) [reply docs/manual/cli/migrate.en.md]: From https://docs.contao.org/dev/reference/commands/#contao-migrate
Why delete the option --no-interaction?

Comment (fritzmg) [reply docs/manual/cli/migrate.en.md]: > Why delete the option --no-interaction?

It's not deleted.

**Review by netzarbeiter — APPROVED** [review]

**Review by fritzmg — APPROVED** [review]

**Pull request merged**